src/pages/Background/index.ts

```diff
@@ -87,6 +90,19 @@ chrome.runtime.onMessage.addListener(async function (
       code: request.code,
       actions: request.actions,
     });
+  } else if (request.type === 'cypress-trigger-start-recording') {
+    const tabId = sender.tab?.id;
+
+    if (tabId == null) {
```

Comment:
nit: I think we can be more explicit here 
```
if (tabId === undefined)
```

Reply:
Is there any downside leaving it to throw on nullish tabIds?

Reply:
`null` and `undefined` sometimes can have different implications. In this case, I don't think there is any downside since null tabId will be thrown somewhere anyway